[
  {
    "message": "folly/test/Test.cpp:6\nValue of: 0\nExpected: 1",
    "stacktrace": "",
    "stdErr": "",
    "stdOut": "hello world\nfolly/test/Test.cpp:6: Failure\nValue of: 0\nExpected: 1",
    "testCase": "BAR",
    "testCaseName": "FOO",
    "time": 0,
    "type": "FAILURE"
  }
]
